code/modules/mob/living/brain/brain_item.dm

```diff
@@ -268,7 +268,7 @@
 
 //Cure a random trauma of a certain resilience level
```

Comment:
Not actually random, but in the order the traumas were added. Should I change the comment or make it actually random?

Reply:
@XDTM

Reply:
Yeah, it'd be better,  thanks